frontend/Dockerfile

```diff
@@ -3,5 +3,6 @@ WORKDIR /usr/src/app/frontend
 COPY package*.json ./
 RUN npm install
 COPY . .
+RUN npm run build
```

Comment:
Technically we can delete everything other than the `build` directory to reduce the size of the image. But this is fine too.